editor/scene_tree_editor.cpp

```diff
@@ -614,21 +614,31 @@ bool SceneTreeEditor::_update_filter(TreeItem *p_parent, bool p_scroll_to_select
 
 	if (!p_parent) {
 		// Tree is empty, nothing to do here.
-		return false;
+		return 0;
```

Comment:
Why? This method returns `bool`, this was correct.

Reply:
Oops. Accidental leftover.